packages/core/src/js/client.ts

```diff
@@ -1,4 +1,4 @@
-import { eventFromException, eventFromMessage } from '@sentry/browser';
```

Comment:
Let's use the export from `@sentry/core`, it's more safe as core should not take any browser assumptions. (In this case it's the same function, but it's a good practice for RN)

Reply:
Updated with 3e36c6dcfcc20c32f8d0d0bc5585cf1d6dd4df30
Note that `eventFromException` is only exported from '@sentry/browser'

Reply:
Reverted the change since it resulted in [a build error](https://github.com/getsentry/sentry-react-native/actions/runs/12070129197/job/33658926517). I suggest to revisit this separately.

Reply:
Yeah, should work, but the implementation are sadly not the exactly same like in case of the `captureFeedback`. :/